docs/reference/indices/create-index.asciidoc

```diff
@@ -78,7 +78,7 @@ that can be set when creating an index, please check the
 [[mappings]]
 === Mappings
 
-The create index API allows to provide a type mapping:
+The create index API allows to provide a mapping:
```

Comment:
The JSON property is still called "mappings", plural so I'd be tempted to describe them as "field mappings".

Reply:
👍